PointerParameters.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#define Size 10
+
+int sum(int *start, int *end)
+{
+   int initialization = 0;
+   while(start < end)
+   {
+      initialization += *start;
+   }
+   return initialization;
+}
+
+int main(void)
+{
+
+int Array[Size]= {20,10,5,39,4,16,19,26,31,20};
```

Comment:
```suggestion
int Array[]= {20,10,5,39,4,16,19,26,31,20};
```

If you remove the size of the array, it will automatically be given a size that fits all of the elements on the right side of the `=`.